jenkins/job-configs/global.yaml

```diff
@@ -7,6 +7,10 @@
                     mkdir -p _tmp
                     curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh" > ./_tmp/upload-to-gcs.sh
                     chmod +x ./_tmp/upload-to-gcs.sh
+
+                    if [[ ! -e "${WORKSPACE}/build-log.txt" ]]; then
+                      curl -fsS --retry 3 "jenkins-master:8080/job/${JOB_NAME}/${BUILD_NUMBER}/consoleText" > "${WORKSPACE}/build-log.txt"
```

Comment:
Yep. The old GCS plugin didn't either. I'll have to make some change to get it to work there, but I think this is a reasonable first step.